mne/viz/topomap.py

```diff
@@ -1203,13 +1209,9 @@ def plot_tfr_topomap(tfr, tmin=None, tmax=None, fmin=None, fmax=None,
     if fmax is not None:
         ifmax = idx[-1] + 1
 
-    data = data[picks, ifmin:ifmax, itmin:itmax]
+    data = data[:, ifmin:ifmax, itmin:itmax]
     data = np.mean(np.mean(data, axis=2), axis=1)[:, np.newaxis]
 
-    if merge_grads:
-        from ..channels.layout import _merge_grad_data
-        data = _merge_grad_data(data)
-
```

Comment:
See, exactly the same code as in 1189.

Reply:
it was moved up, this part was removed.

Reply:
...

Can we just ignore this happened?